_drafts/shortcut-long-pain.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Shortcut, Long Pain
+---
+
+Programmers are often tempted to take the shortest route possible to deliver a
+feature. While there is a lot to be said about pragmatism, you should be
+considerate of consequences to make the best decision possible.
```

Comment:
I think the wording of this sentence can be improved upon. It's not entirely clear to me. Perhaps something along the lines of:

```
While there is a lot to be said about pragmatism, every decision you make has consequences. So, make the best decision possible
```